modules/secret/main.tf

```diff
@@ -9,6 +9,14 @@ resource "google_secret_manager_secret" "this" {
   }
 }
 
+// Create a dummy GCP secret version to avoid bad reference on first deploy.
+resource "google_secret_manager_secret_version" "dummy" {
```

Comment:
You need lifecycle annotations or this will reconcile things away

Reply:
added